llvm/tools/obj2yaml/xcoff2yaml.cpp

```diff
@@ -204,22 +213,19 @@ Error XCOFFDumper::dumpSymbols() {
     Sym.NumberOfAuxEntries = NumOfAuxSym;
 
     if (NumOfAuxSym) {
-      auto AuxEntTbl = &Sym.AuxEntries;
+      std::vector<std::unique_ptr<XCOFFYAML::AuxSymbolEnt>> AuxEntTbl;
```

Comment:
I think you can just do `std::vector<std::unique_ptr<XCOFFYAML::AuxSymbolEnt>> &AuxEntTbl = Sym.AuxEntries;`. Then you don't need to move at the end of the block.